src/security.rst

```diff
@@ -0,0 +1,74 @@
+Security
+========
+
+TODO: Fill in other parts of this document, with information about
+cylc-flow (jobs, zmq, etc), UI Server, etc.
+
+Web
+~~~
```

Comment:
RST lets you use a whole load of characters for different section levels, Sphinx associates these in the order it comes across them.

For ease of editing/refactoring there is a loose convention for Sphinx sectioning.

```
Title
====

Heading
-------

Sub-Heading
^^^^^^^^^^^

Sub-Sub-Heading
""""""""""""""
When these appear it's time to refactor.
```

(note: we've got a documentation writers reference thinggy in the Rose docs - http://metomi.github.io/rose/doc/html/developing/restructuredtext.html#headings)

Reply:
Done in a new commit